README.md

```diff
@@ -93,41 +99,61 @@ create-ns: Success, created nsid:6
 ```
 5. Attach a namespace to a controller
 ```
-#sudo ./switchtec-nvme attach-ns 0x3300@/dev/switchtec0 -n 6 -c 0x21
+#sudo ./switchtec-nvme attach-ns 0x3300@/dev/switchtec0 -n 6 -c 1
 attach-ns: Success, nsid:6
 ```
 6. List all NVMe devices and namespaces
 ```
 #sudo ./switchtec-nvme switchtec list
 Node                       SN                   Model                                    Namespace Usage                      Format           FW Rev
 -------------------------- -------------------- ---------------------------------------- --------- -------------------------- ---------------- --------
-0x3300n1@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             1           6.55  MB /   6.55  MB    512   B +  0 B   REVISION
-0x3300n2@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             2          13.11  MB /  13.11  MB    512   B +  0 B   REVISION
-0x3300n3@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             3         131.07  MB / 131.07  MB    512   B +  0 B   REVISION
-0x3300n4@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             4           2.15  GB /   2.15  GB    512   B +  0 B   REVISION
-0x3300n5@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             5           2.15  GB /   2.15  GB    512   B +  0 B   REVISION
-0x3300n6@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             6           4.29  GB /   4.29  GB      4 KiB +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             1           6.55  MB /   6.55  MB    512   B +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             2          13.11  MB /  13.11  MB    512   B +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             3         131.07  MB / 131.07  MB    512   B +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             4           2.15  GB /   2.15  GB    512   B +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             5           2.15  GB /   2.15  GB    512   B +  0 B   REVISION
+0x3300@/dev/switchtec0   SERIALNUMBER         VENDOR MODEL                             6           4.29  GB /   4.29  GB      4 KiB +  0 B   REVISION
 ```
-7. Detach a namespace from a controller
+7. Allocate VQ and VI resources for the controller
+```
+#sudo ./switchtec-nvme virt-mgmt 0x3300@/dev/switchtec0 -c1 -r0 -a8 -n2
+success, Number of Resources allocated:0x2
+#sudo ./switchtec-nvme virt-mgmt 0x3300@/dev/switchtec0 -c1 -r1 -a8 -n2
+success, Number of Resources allocated:0x2
+```
+8. Reset the controller and then bring it online
+```
+#sudo ./switchtec-nvme switchtec vf-reset 0x3301@/dev/switchtec0 -f
```

Comment:
We should add some description about how we know there's a 0x3301 which is a VF.